packages/node/src/app/analytics-node.ts

```diff
@@ -33,25 +33,26 @@ export class Analytics extends NodeEmitter implements CoreAnalytics {
 
   constructor(settings: AnalyticsSettings) {
     super()
+    settings.flushAt ??= settings.maxEventsInBatch ??= 15
```

Comment:
Wondering if we should create a copy of these settings - typically we don't want to mutate an object the user passes in.

Reply:
I totally agree with this.